operations/sat/sat_in_csm.md

```diff
@@ -0,0 +1,59 @@
+# System Admin Toolkit (SAT) in CSM
+
+The System Admin Toolkit (SAT) is a command-line interface that can assist administrators with common tasks, such as
+troubleshooting and querying information about the HPE Cray EX System, system boot and shutdown, and replacing hardware
+components. In CSM 1.3 and newer, the `sat` command is available on the Kubernetes NCNs without installing the SAT
+product stream.
+
+## SAT product stream components
+
+It is still possible to install SAT as a separate product stream. Any version of SAT installed as a separate product
+stream overrides the `sat` command available in CSM. Installing the SAT product stream allows additional supporting
+components to be added:
+
+- An entry for SAT in the `cray-product-catalog` Kubernetes ConfigMap is only created by installing the SAT product
+  stream. Otherwise, there will be no entry for this version of SAT in the output of `sat showrev`.
+
+- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
+  provides uninstall and activate functionality when used with the `prodmgr` command. (In SAT 2.3 and older, SAT was
+  only available to install as a separate product stream. Because these versions were packaged with
+  `sat-install-utility`, it is still possible to uninstall these versions of SAT.)
+
+- The `docs-sat` RPM package is only available with the full SAT product stream. You can find SAT documentation at
+  the links below (see [SAT documentation](#sat-documentation)).
+
+- The `sat-config-management` git repository in Gitea (VCS) and thus the SAT layer of NCN CFS configuration is
+  only available with the full SAT product stream.
+
```

Comment:
I recommend adding the heading "CFS configuration" back.

If we create a heading level, we want to make sure there is more than one. That way it doesn't oddly stick out on its own when there might be a better way to present the information. In your draft of this change for the SAT docs, the "CFS configuration" heading was on its own, so it was better to remove it. Here, though, we also have a second-level heading (##) of "SAT documentation," so I recommend adding "CFS configuration" back as a heading. That way we have two second-level headings, and "SAT documentation" isn't on its own.

That way there are two second-level headings for this section (not a single heading like in the SAT docs where we didn't have to link to the SAT documentation).

Reply:
I feel like in both cases it's supposed to be a child of "SAT product stream components" though, in that the SAT CFS configuration is one of the product stream components not available unless you install the full product stream. For that reason I originally had it as a 3rd-level heading in both locations - though as you pointed out we want to avoid having a new level of heading if there is only one.

> That way we have two second-level headings, and "SAT documentation" isn't on its own.

Right now we have two level-2 headings on this page, "SAT product stream components" and "SAT Documentation", right?

Reply:
Man, my eyes are playing tricks on me. I agree with you, and you can disregard this comment :)